client/state/posts/revisions/selectors.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { get, map } from 'lodash';
+import createSelector from 'lib/create-selector';
+
+/**
+ * Internal dependencies
+ */
+import { normalizePostForDisplay } from '../utils';
+
+export const getPostRevisions = createSelector(
+	( state, siteId, postId ) => {
+		return map(
+			get( state.posts.revisions.revisions, [ siteId, postId ], [] ),
+			normalizePostForDisplay );
```

Comment:
np, I wonder if it's worth maybe updating the `if` example in the [Javascript Guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#multi-line-statements)?

Currently it's showing: 
```javascript
if ( firstCondition() && secondCondition() &&
		thirdCondition() ) {
	doStuff();
}
```